cli/src/index.ts

```diff
@@ -1,8 +1,31 @@
 #!/usr/bin/env node
 
 import * as dotenv from 'dotenv';
+import pc from 'picocolors';
+import boxen from 'boxen';
 import program from './commands/index.js';
 
 dotenv.config();
 
-await program.parseAsync(process.argv);
+try {
+  await program.parseAsync(process.argv);
+} catch (e: any) {
+  console.error(pc.red(`An unexpected error occurred with code ${e.code} and message → ${e.message}`));
```

Comment:
This will hide the stack-trace which make it hard for reporting.

Reply:
Changed